mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -675,6 +675,12 @@ func schemaNodeConfig() *schema.Schema {
 						},
 					},
 				},
+				"resource_manager_tags": {
```

Comment:
Can you please add this field to the `resource_container_node_pool` documentation file as well?

Reply:
this references the resource_container_clusters node_config block so no need